app/views/dashboard/get.html.erb

```diff
@@ -15,3 +15,12 @@
     <% end %>
   </table>
 <% end %>
+<br>
+<br>
+New Semester
+<form method="POST">
```

Comment:
Maybe we should use Ruby form helpers here. Thoughts?

http://guides.rubyonrails.org/form_helpers.html